File: mobile/android/QuestionRepository.kt
package com.desbloqueiointeligente

data class EducationQuestion(
    val year: String,
    val age: Int,
    val subject: String,
    val text: String,
    val alternatives: List<String>,
    val correctAnswer: String,
    val level: String,
    val timeSeconds: Int
)

object QuestionRepository {
    val defaultQuestions = listOf(
        EducationQuestion("1º", 6, "Matemática", "Quanto é 2 + 1?", listOf("2", "3", "4"), "B", "Fácil", 10),
        EducationQuestion("2º", 7, "Português", "Qual palavra começa com B?", listOf("Bola", "Casa", "Dedo"), "A", "Fácil", 10),
        EducationQuestion("3º", 8, "Lógica", "Qual número vem depois do 9?", listOf("8", "10", "7"), "B", "Fácil", 10),
        EducationQuestion("4º", 9, "Ciências básicas", "As plantas precisam de luz?", listOf("Sim", "Não", "Nunca"), "A", "Fácil", 10),
        EducationQuestion("5º", 10, "Matemática", "Quanto é 20 dividido por 4?", listOf("4", "5", "6"), "B", "Médio", 15)
    )
}

File: mobile/ios/QuestionRepository.swift
import Foundation

enum QuestionRepository {
    static let defaultQuestions = [
        EducationQuestion(year: "1º", age: 6, subject: "Matemática", text: "Quanto é 2 + 1?", alternatives: ["2", "3", "4"], correctAnswer: "B", level: "Fácil", timeSeconds: 10),
        EducationQuestion(year: "2º", age: 7, subject: "Português", text: "Qual palavra começa com B?", alternatives: ["Bola", "Casa", "Dedo"], correctAnswer: "A", level: "Fácil", timeSeconds: 10),
        EducationQuestion(year: "3º", age: 8, subject: "Lógica", text: "Qual número vem depois do 9?", alternatives: ["8", "10", "7"], correctAnswer: "B", level: "Fácil", timeSeconds: 10),
        EducationQuestion(year: "4º", age: 9, subject: "Ciências básicas", text: "As plantas precisam de luz?", alternatives: ["Sim", "Não", "Nunca"], correctAnswer: "A", level: "Fácil", timeSeconds: 10),
        EducationQuestion(year: "5º", age: 10, subject: "Matemática", text: "Quanto é 20 dividido por 4?", alternatives: ["4", "5", "6"], correctAnswer: "B", level: "Médio", timeSeconds: 15)
    ]
}
